invoke-ai/InvokeAI
# Fix function call that we forgot to update in #6606

## Summary

Fix function call that we forgot to update in #6606

## QA Instructions

Run a TiledMultiDiffusionDenoiseLatents invocation and make sure it doesn't crash.

## Checklist
